.github/CODEOWNERS

```
@@ -0,0 +1,12 @@
+# Modifications to the source code should be handled by the code review team
+*.java                             @MockBukkit/code-reviewers
```

This team got renamed
```suggestion
*.java                             @MockBukkit/project-members
```